src/traits.rs

```diff
@@ -774,10 +774,40 @@ pub trait SystemExt: Sized + Debug + Default + Send + Sync {
     /// use sysinfo::{System, SystemExt};
     ///
     /// let mut s = System::new_all();
+    /// s.refresh_cpu_usage();
+    /// ```
+    fn refresh_cpu_usage(&mut self) {
+        self.refresh_cpu_specifics(CpuRefreshKind::new().with_cpu_usage())
+    }
+
+    /// Refreshes CPUs frequency information.
+    ///
+    /// Calling this method is the same as calling
+    /// `refresh_cpu_specifics(CpuRefreshKind::new().with_frequency())`.
+    ///
+    /// ```no_run
+    /// use sysinfo::{System, SystemExt};
+    ///
+    /// let mut s = System::new_all();
+    /// s.refresh_cpu_frequency();
+    /// ```
+    fn refresh_cpu_frequency(&mut self) {
+        self.refresh_cpu_specifics(CpuRefreshKind::new().with_frequency())
+    }
+
+    /// Refreshes all information related to CPUs information.
+    ///
+    /// Calling this method is the same as calling
+    /// `refresh_cpu_specifics(CpuRefreshKind::everything())`.
+    ///
+    /// ```no_run
+    /// use sysinfo::{System, SystemExt};
+    ///
+    /// let mut s = System::new_all();
     /// s.refresh_cpu();
     /// ```
     fn refresh_cpu(&mut self) {
```

Comment:
Please copy the paragraph `⚠️ Please note that the result will very likely be inaccurate at the first call...` for this method as well.